src/View/Widget/WidgetLocator.php

```diff
@@ -36,7 +36,7 @@
  *
  * Widgets can ask for the current view by using the `_view` widget.
  */
-class WidgetRegistry
+class WidgetLocator
```

Comment:
You need to keep the old class for BC.

Reply:
There is an alias at the end of the file

Reply:
Aliases have to be in the original file so that if the old alias is used first it will work.

Reply:
Done, used the same file as `Cake\Network\Http\Auth\Basic` => `Cake\Http\Client\Auth\Basic`